release-content/migration_guides_template.md

```diff
@@ -17,10 +17,19 @@ Aim to communicate:
 
 Keep it short and sweet:
 
-- Use bullet points and make sure it's searchable.
+- What, then why, then how to migrate.
+- Some helpful standardized phrases:
+  - `OldType` is now `NewType`. Replace all references and imports.
+  - The `Struct::method()` method now requires an additional `magnitude: f32` argument.
+  - `Enum` has a new variant, `Enum::NewVariant`, which must be handled during `match` statements.
+  - The `Type::method` method has been removed. Use `Type::other_method` instead.
+  - The `crate::old_module` module is now `crate::new_module`. Update your imports.
+  - `function` now returns `Option<String>`, instead of `String`.
+- Make sure it's searchable by directly naming the types and methods involved.
+- Use backticks for types, methods and modules (e.g. `Vec<T>` or `core::mem::swap`).
+- Use bullet points to explain complex changes.
 - Avoid headings. If you must, use only level-two headings.
```

Comment:
It's unrelated to this PR, but I though @NthTensor said this would be handled with tooling. So we don't impose any heading level, the user start using H1, then H2, etc. and the export/import tool will convert to the proper level. The only limit we should impose is "don't use more than X levels" (I think it was worded like that at some point).

Reply:
The markdown linter wants us to use level-2 because of the title in the front matter. No idea how to change that.